client/js/router/index.js

```diff
@@ -5,22 +5,25 @@ import LoginSignup from '../views/LoginSignup.vue';
 import Lobby from '../views/Lobby.vue';
 import GameView from '../views/GameView.vue';
 import Rules from '../views/Rules.vue';
+import Stats from '../views/Stats.vue';
 import store from '../store/store.js';
 
 Vue.use(VueRouter);
 
+const mustBeAuthenticated = (to, from, next) => {
+  if (store.state.auth.authenticated) {
+    next();
+  } else {
+    next('/login');
+  }
+};
```

Comment:
Moved this func to outside the Home component for general reuse. Could consider a separate utility file for it, but it's still a small list of routes so I just put it at the top

Reply:
Seems like a good candidate for a middleware utility on the vue router.